controllers/admin/AdminDPDBalticsSettingsController.php

```diff
@@ -114,6 +114,21 @@ protected function initOptions()
                     'title' => $this->l('Save'),
                 ],
             ],
+            'orders_page_configuration' => [
+                'title' => $this->l('Orders listing page configuration'),
```

Comment:
missing FILE_NAME and use module. 

$this->module->l('Save', self::FILE_NAME),